tsconfig.json

```diff
@@ -1,6 +1,14 @@
 {
   "compilerOptions": {
-    "target": "es2020",
+    // https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping
+    // https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12
+    "lib": [
+      "es2019",
```

Comment:
i would just set this as es2020, as the latest 12.x supports all of it afaik

Reply:
or at least thats what some quick googling told me 😂  🙊

Reply:
Agreed, but that isn't until v12.9+. I figured it would be better to just support all of v12 and use this instead.